### PR TITLE
Feat/author profile animations

### PR DESCRIPTION
**Pull request by jimhunty**
This PR includes

- Adding animations to the styleguide with a web and native Fade In.
- Applying those animations for Author Profile
- Adding a test and story for the new animations
- Exposing keyframes from `responsive-styles` to ensure our `styled-components` abstraction remains.
- An long term strategy for Styleguide

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/8b770d9ce03c1fd487df01ab204feda08a63b98a/dextrose-test/index.html

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "scripts": {
     "native": "node node_modules/react-native/local-cli/cli.js start",
     "test": "yarn storybook:test-build && jest ./lib --ci --bail --coverage",
+    "test:local": "yarn test && npx lerna run test:all --since --stream",
```
Comment (craigbilner) [inline package.json]: don't need `npx` here and could be `test:all`?

File: packages/styleguide/package.json [hunk 2]
```diff
@@ -7,7 +7,10 @@
     "fmt": "prettier --write '**/*.*'",
     "prettier:diff": "prettier --list-different '**/*.*'",
     "lint": "eslint . && npm run prettier:diff",
-    "test": "jest --config='./__tests__/jest.config.js'"
+    "test:all": "yarn test:android --ci --bail --coverage && yarn test:ios --ci --bail --coverage && yarn test:web --ci --bail --coverage",
```
Comment (craigbilner) [inline packages/styleguide/package.json]: needs rebasing

File: packages/styleguide/styleguide.js [hunk 3]
```diff
@@ -1 +1,10 @@
-export { default } from "./colours/section";
+import sectionColours from "./colours/section";
+import FadeIn from "./animations";
+
+export const Colours = {
```
Comment (craigbilner) [inline packages/styleguide/styleguide.js]: capital C?

File: packages/article-label/article-label.stories.js [hunk 4]
```diff
@@ -3,12 +3,12 @@ import React from "react";
...
 import invert from "lodash.invert";
-import sectionColours from "@times-components/styleguide";
+import { Colours } from "@times-components/styleguide";
 import ArticleLabel from "./article-label";
 
 storiesOf("Primitives/ArticleLabel", module).add("ArticleLabel", () => (
   <ArticleLabel
     title="swimming"
-    color={select("Section", invert(sectionColours), "#333333")}
+    color={select("Section", invert(Colours.sectionColours), "#333333")}
```
Comment (craigbilner) [inline packages/article-label/article-label.stories.js]: default colour?

**Pull request merged**